datafusion/functions/Cargo.toml

```diff
@@ -52,7 +52,7 @@ encoding_expressions = ["base64", "hex"]
 # enable math functions
 math_expressions = []
 # enable regular expressions
-regex_expressions = ["regex"]
+regex_expressions = ["regex", "string_expressions"]
```

Comment:
Have to do this for `StringArrayType`. Maybe we should consider relocating it to a common package?

Reply:
I would be very open to moving the StringArrayType, StringArrayBuilder, StringViewArrayBuilder, etx to a file in functions such as string_array.rs as they are used in multiple modules (unicode, regex, likely datetime in the future, etc) and are quite useful in general for any external UDF that might need them.